core/schema/directory.go

```diff
@@ -34,7 +34,7 @@ func (s *directorySchema) Resolvers() router.Resolvers {
 		"Query": router.ObjectResolver{
 			"directory": router.ToResolver(s.directory),
 		},
-		"Directory": router.ObjectResolver{
+		"Directory": router.ToIDableObjectResolver(core.DirectoryID.ToDirectory, router.ObjectResolver{
```

Comment:
TIL parentheses are optional in `(core.DirectoryID).ToDirectory`

Reply:
Yeah I almost didn't believe this when my IDE didn't report any errors, but it worked!